docs/guides/restart_job.md

```diff
@@ -53,3 +53,32 @@ $ floyd restart mckay/projects/mnist/1 --gpu
 # Run the job with a new version of a dataset
 $ floyd restart mckay/projects/mnist/1 --data mckay/datasets/mnist/2:mnist
 ```
+
+## Parameter sweeping:
+
+A great use case for the restart command is when you need to run a series of jobs with 
+different training parameters. You just need to make your training script take in all the 
+parameters from command line. After you run the first job using the floyd run command:
```

Comment:
"from command line" should be "from the command line"

"floyd run" should be `floyd run`